msgcntr/messageforums-app/src/webapp/jsp/discussionForum/statistics/dfStatisticsFullTextForOne.jsp

```diff
@@ -111,9 +112,9 @@
 						</h:column>
 					</h:dataTable>
 				</h:panelGroup>
-  			</h:column>
```

Comment:
Looks like this tag is not closed, is it?

Reply:
Yes, it is closed on line 145.  This page had a datatable with two columns, of which only one would render based on stat.msgDeleted.  Instead, I condensed it to one column and put the content in two panelGroups.  The 2nd column is removed at line 116.  Otherwise there's always be an empty column rendered.